doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -170,6 +174,8 @@ Common Application Framework (CAF)
     ``MODULE_ID`` macro and :c:func:`module_id_get` function now returns module reference from dedicated section instead of module name.
     The module name can not be obtained from reference object directly, a helper function (:c:func:`module_name_get`) should be used instead.
 
+* Added possibility of setting custom Bluetooth LE advertising intervals.
```

Comment:
Was this added to a module or to CAF in general? If to a module, add it like the entries in lines 166-169. If to CAF in general, move the entry to after line 165.

Reply:
This is not fixed. Please fix this.

Reply:
I moved it now. Please see if it's ok.